internal/defaults/defaults.go

```diff
@@ -17,60 +17,140 @@ package defaults
 import "time"
 
 const (
+	// Debug enables debug messages.
 	Debug = false
 
-	CAGenerate         = false
-	CAReuseSecret      = false
-	CACommonName       = "Cilium CA"
-	CAValidityDuration = 3 * 365 * 24 * time.Hour
-	CASecretName       = "cilium-ca"
+	// CiliumNamespace is the Kubernetes namespace in which Cilium is
+	// installed.
+	CiliumNamespace = "kube-system"
 
-	HubbleServerCertGenerate         = false
-	HubbleServerCertCommonName       = "*.default.hubble-grpc.cilium.io"
-	HubbleServerCertValidityDuration = 3 * 365 * 24 * time.Hour
-	HubbleServerCertSecretName       = "hubble-server-certs"
+	// K8sRequestTimeout specifies the timeout for K8s API requests.
+	K8sRequestTimeout = 60 * time.Second
 
-	HubbleRelayServerCertGenerate         = false
-	HubbleRelayServerCertCommonName       = "*.hubble-relay.cilium.io"
+	// CAGenerate can be set to true to generate a new Cilium CA secret.
+	// If CAReuseSecret is true, then a new CA secret only is created if
+	// existing one is not found.
+	CAGenerate = false
+	// CAReuseSecret can be set to true to store and load the Cilium CA from
+	// the secret if it exists. Setting to false will delete the old Secret and
+	// force regeneration.
+	CAReuseSecret = false
+	// CACommonName is the Cilium CA x509 certificate CN value.
+	CACommonName = "Cilium CA"
+	// CAValidityDuration represent how much time the Cilium CA certificate
+	// generated by certgen is valid.
+	CAValidityDuration = 3 * 365 * 24 * time.Hour
+	// CASecretName is the Kubernetes Secret in which the Cilium CA certificate
+	// is read from and/or written to.
+	CASecretName = "cilium-ca"
+
+	// HubbleServerCertGenerate can be set to true to generate and store a
+	// Hubble server TLS certificate.
+	HubbleServerCertGenerate = false
+	// HubbleServerCertCommonName is the Hubble server x509 certificate CN
+	// value (also used as DNS SAN).
+	HubbleServerCertCommonName = "*.default.hubble-grpc.cilium.io"
+	// HubbleServerCertValidityDuration represent how much time the Hubble
+	// server certificate generated by certgen is valid.
+	HubbleServerCertValidityDuration = 3 * 365 * 24 * time.Hour
+	// HubbleServerCertSecretName is the Kubernetes Secret in which the Hubble
+	// server certificate is written to.
+	HubbleServerCertSecretName = "hubble-server-certs" //#nosec
```

Comment:
What was the linter complaining about here?

Reply:
If you are asking about the `//#nosec`, it's because of

```
internal/defaults/defaults.go:58:2: G101: Potential hardcoded credentials (gosec)
	HubbleServerCertSecretName = "hubble-server-certs"
```